ADR/0022-secret-mgmt-for-user-workloads.md

```diff
@@ -380,6 +380,26 @@ status:
     status: "False"
     type: DataObtained
 ```
+### Internal Secrets
+
+To distinguish secrets that were provided by end-users from secrets that are generated by RHTAP's components, a label `appstudio.redhat.com/internal: true` is introduced. The presence of this label indicates that the secret is intended for internal usage, and unauthorized changes to such secrets may lead to unpredictable results.
```

Comment:
This is intended to only be used for RemoteSecrets. While that is implied because of the ADR that this is in, can we be more explicit about the scope to ensure that it isn't extrapolated out of context?